app/api/connectors/httpparsers/HttpParser.scala

```diff
@@ -16,14 +16,14 @@
 
 package api.connectors.httpparsers
 
-import api.models.errors.{BVRError, DownstreamError, DownstreamErrorCode, DownstreamErrors, InternalError, OutboundError}
+import api.models.errors._
+import play.api.libs.Files.logger
 import play.api.libs.json._
 import uk.gov.hmrc.http.HttpResponse
-import utils.Logging
 
 import scala.util.{Success, Try}
 
-trait HttpParser extends Logging {
+trait HttpParser {
 
```

Comment:
need to add `  private val logger: Logger = Logger(this.getClass)` here and then update line 20 to be `import play.api.Logger`